packages/pwa-kit-create-app/scripts/create-mobify-app-dev.js

```diff
@@ -120,9 +123,13 @@ const withLocalNPMRepo = (func) => {
 const runGenerator = () => {
     // Shelljs can't run interactive programs, so we have to switch to child_process.
     // See https://github.com/shelljs/shelljs/wiki/FAQ#running-interactive-programs-with-exec
-    const cmd = 'npx'
-    const args = ['pwa-kit-create-app', ...process.argv.slice(2)]
-    cp.execFileSync(cmd, args, {stdio: 'inherit'})
+    try {
+        cp.execSync(`npx pwa-kit-create-app --outputDir ${process.argv.slice(3)}`, {
+            stdio: 'inherit'
+        })
+    } catch (error) {
+        console.log(`Error status: ${error.status} with '${error.message}'`)
```

Comment:
@adamraya Please don't catch and swallow errors like this. The old code crashed on error, the new code must also crash on error – not fail silently, with a log message. Super important!

Reply:
Thanks @olibrook. The catch block was added specifically for debugging purposes only to force the error message to be printed in the terminal, without it, the error message was not visible in the terminal of the Circle CI machine. 

The try/catch is gone now and will throw if an error exists and fail the build as expected.